website/content/en/docs/concepts/fabric.md

```diff
@@ -28,15 +28,15 @@ The following assume that you have access to the `clusterlink` CLI and one or mo
  peers (i.e., clusters) where you'll deploy ClusterLink. The CLI can be downloaded
  from the ClusterLink [releases page on GitHub](https://github.com/clusterlink-net/clusterlink/releases/latest).
 
-### Create a new Fabric CA
+### Create a new fabric CA
 
-To create a new Fabric certificate authority (CA), execute the following CLI command:
+To create a new fabric certificate authority (CA), execute the following CLI command:
 
 ```sh
 clusterlink create fabric --name <fabric_name>
 ```
 
-This command will create the CA files `cert.pem` and `key.pem` in a folder named <fabric_name>. The `--name` option is optional, and by default, "default_fabric" will be used. While you will need access to these files to create the peers` gateway certificates later, the private key file should be protected and not shared with others.
+This command will create the CA files `cert.pem` and `key.pem` in a directory named <fabric_name>. The `--name` option is optional, and by default, "default_fabric" will be used. While you will need access to these files to create the peers` gateway certificates later, the private key file should be protected and not shared with others.
```

Comment:
nit: can you please break this line into multiple lines in the markdown, at about the same length as the rest of the file?